MapleServer2/Types/Party.cs

```diff
@@ -0,0 +1,70 @@
+﻿using MaplePacketLib2.Tools;
+using MapleServer2.Servers.Game;
+using System;
+using System.Collections.Generic;
+
+namespace MapleServer2.Types
+{
+    public class Party
+    {
+        public int Uid { get; private set; }
+        public int MaxMembers { get; set; }
+        public Player Leader { get; set; }
+
+        //List of players and their session.
+        public HashSet<Player> Players { get; private set; }
```

Comment:
I feel like a better word than `Players` here is `Members` and then the methods are `AddMember`, etc...